src/traces/funnel/attributes.js

```diff
@@ -23,6 +24,9 @@ module.exports = {
 
     hovertext: barAttrs.hovertext,
     hovertemplate: barAttrs.hovertemplate,
+    hoverinfo: extendFlat({}, plotAttrs.hoverinfo, {
+        flags: ['percentInitial', 'percentPrevious', 'percentTotal'].concat(plotAttrs.hoverinfo.flags)
```

Comment:
Nop. These should be `'percent initial'` ... just like for `textinfo`

So if I wasn't clear yesterday.